source/_posts/2024-02-07-release-20242.markdown

```diff
@@ -156,6 +158,32 @@ the new "assist devices" button.
 
 {% my voice_assistants badge %}
 
+## Better errors when using Assist
+
+If you are a user of [Assist](https://home-assistant.io/voice_control/), our private voice assistant, you may have noticed that its responses were not meaningful when something was not understood.
+
+Some of the errors encountered while using Assist can easily be fixed on your side by adding aliases to entities or areas, exposing entities to Assist, or assigning entities and devices to the correct areas.
+
+So we are taking the first step to help you fix these errors: as of this release, Assist provides much better errors in case your intention is understood, but something else is missing (An unknown name, area, device class, or domain)
+
+<img class="no-shadow" src='/images/blog/2024-02/assist-errors.png' alt='Screenshot showing the new errors assist will return in case the intention is understood, but something else is missing.'>
+
+## Custom response in the automation editor when using a sentence trigger
+
+As almost every component and feature of Home Assistant, Assist can be customized and extended to understand more sentences. 
+
+A few releases ago, we introduced a very simple way to extend what Assist understands: The [sentence trigger](https://www.home-assistant.io/docs/automation/trigger/#sentence-trigger) in our automation engine.
+
+Up until now, using a sentence trigger always led to the same Assist response: “Done”. In fact, the only way to define a custom response was to write complex custom sentences in YAML.
+
+In this release, we introduce a new action to set a custom response in your sentence-triggered automations directly inside the automation editor.
+
+<img class="no-shadow" src='/images/blog/2024-02/assist-custom-response-editor.png' alt='Screenshot showing how to build a custom response in our automation editor.'>
+
+The response field accepts templates, so it can be used to build complex responses, for example listing all your room temperatures.
```

Comment:
```suggestion
The response field accepts templates, so it can be used to build complex responses, for example, listing all your room temperatures.
```